openlibrary/core/vendors.py

```diff
@@ -141,20 +202,19 @@ def _get_amazon_metadata(id_, id_type='isbn'):
     :rtype: dict or None
     """
 
-    kwargs = {}
     if id_type == 'isbn':
         id_ = normalize_isbn(id_)
-        kwargs = {'SearchIndex': 'Books', 'IdType': 'ISBN'}
-    kwargs['ItemId'] = id_
-    kwargs['MerchantId'] = 'Amazon'  # Only affects Offers Response Group, does Amazon sell this directly?
-
-    if not lending.amazon_api:
-        raise Exception("Open Library is not configured to access Amazon's API")
-    product = lending.amazon_api.lookup(**kwargs)
-    # sometimes more than one product can be returned, choose first
-    if isinstance(product, list):
-        product = product[0]
-    return _serialize_amazon_product(product)
+    try:
+        if not lending.amazon_api:
+            raise Exception
+        product = lending.amazon_api.get_product(id_)
+        # sometimes more than one product can be returned, choose first
+        if isinstance(product, list):
+            product = product[0]
+    except Exception as e:
+        return None
```

Comment:
This undoes the work I did to log API issues for debugging. I think it should be removed.

Reply:
We need a way of doing this that will not cause book pages to crash on production; feel free to extend this and please test on dev.

Reply:
Which exception(s) do we _expect_ to see?  We should specify that exception(s) instead of using the blanket `Exception` as discussed in [PEP8](https://www.python.org/dev/peps/pep-0008/#programming-recommendations) and in the a bit overstated https://realpython.com/the-most-diabolical-python-antipattern